howso/utilities/feature_attributes/relational.py

```diff
@@ -820,14 +820,20 @@ def _infer_feature_bounds(self,  # noqa: C901
                     if not feature_attributes[feature_name].get('cycle_length'):
                         raise ValueError(f'Error computing loose bounds for {feature_name}: '
                                          '`cycle_length` must be specified in attributes')
-                    return {'min': 0, 'max': feature_attributes[feature_name]['cycle_length'],
-                            'allow_null': allow_null}
+                    return {
+                        'min': 0, 'max': feature_attributes[feature_name]['cycle_length'],
+                        'actual_min': 0, 'actual_max': feature_attributes[feature_name]['cycle_length'],
```

Comment:
Looks like this file still uses `actual` instead of `observed`

Reply:
Good eye. Thanks. Fixed.